controllers/upgrade_handlers.go

```
@@ -54,9 +59,86 @@ func (b BackupValidationError) Error() string {
 
 // handleUpgrade orchestrate main upgrade steps and update status as needed
 func (r *ImageBasedUpgradeReconciler) handleUpgrade(ctx context.Context, ibu *lcav1alpha1.ImageBasedUpgrade) (ctrl.Result, error) {
-
 	r.Log.Info("Starting handleUpgrade")
 
+	currentStaterootName, _ := r.RPMOstreeClient.GetCurrentStaterootName()
```

why ignoring the error?

---

Updated